components/integrations/installed_incoming_webhook.tsx

```diff
@@ -102,6 +117,13 @@ export default class InstalledIncomingWebhook extends React.PureComponent<Props>
             );
         }
 
+        const displayEnabled = incomingWebhook.enabled ? null : (
```

Comment:
variable naming is misleading here. It would be better called `disabledSuffix` or similar.

Reply:
on second thought and to circumvent the naming issue you can just add this conditionally in the return value in line 173:

```
{!incomingWebHook.enabled && (
    <FormattedMessage
        id='installed_incoming_webhooks.disabled_webhook'
        defaultMessage=' (Disabled)'
    />
)}